core/domain/rte_component_registry.py

```diff
@@ -64,3 +67,64 @@ def get_tag_list_with_attrs(cls):
                 for ca_spec in component_specs['customization_arg_specs']]
 
         return component_tags
+
+    @classmethod
+    def get_component_types_to_component_classes(cls):
+        """Get component classes mapping for component types."""
+        module_fullname = 'extensions.rich_text_components.components'
```

Comment:
Take a look at how the interaction registry does this. Mimic that instead.

Reply:
Done.